Classes/Bookmark v2/BookmarkNavigationController.swift

```diff
@@ -38,28 +38,26 @@ final class BookmarkNavigationController {
             selector = #selector(BookmarkNavigationController.add(sender:))
         }
 
-        let item = UIBarButtonItem(
-            image: UIImage(named: imageName),
-            style: .plain,
-            target: self,
-            action: selector
-        )
+        let itemBtn = UIButton(frame: Styles.Sizes.barButton)
+        itemBtn.setImage(UIImage(named: imageName)?.withRenderingMode(.alwaysTemplate), for: .normal)
+        itemBtn.addTarget(self, action: selector, for: .touchUpInside)
+        let item = UIBarButtonItem(customView: itemBtn)
```

Comment:
Creating button and adding it as a custom view to bar button item, is kind of used in many places. We could have a simple `convenience init` for `UIBarButtonItem` like below

```swift
extension UIBarButtonItem {
   convenience init(image: UIImage?, target: Any, action: Selector) {
         let button = UIButton(type: .custom)
         button.setImage(image?.withRenderingMode(.alwaysTemplate), for: .normal)
         button.imageView?.tintColor = Styles.Colors.Blue.medium.color
         button.addTarget(target, action: action, for: .touchUpInside)
         button.frame = Styles.Sizes.barButton
         self.init(customView: button)
    }
}
```